content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -14,5 +14,35 @@ match:
 tooltip: >-
   Initialized by running `dvc init` in the **workspace** (typically a Git
   repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  `.dvc` files created with commands such as `dvc add` or `dvc run`.  
+  [More info](/doc/user-guide/basic-concepts/dvc-project)
 ---
+
+# DVC Project
+
+Initialized by running `dvc init` in your <abbr>workspace</abbr> directory, a
+DVC project will contain all of the [DVC metafiles and
+directories][dvc-metafiles], such as the <abbr>cache</abbr>, `dvc.yaml`, and
```

Comment:
cache location can be changed though

Reply:
yes, but explicitly stating that here may seem tangential/distract from the point? I don't have strong opinions here. The `<abbr/>` tooltip will of course provide more details.

Reply:
> cache location can be changed"such as" leaves room for that

Reply:
> cache location can be changed

"such as" leaves room for that